### DISCUSSION TITLE
Copilot is stuck on chat

### DISCUSSION THREAD

**Discussion by georgeani** [community/community#123123] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I got GitHub copilot up and running on my PyCharm last week.

Unfortunately, after 4 conversations I cannot start new conversations and ask any more questions. The suggestions also do not work. Is there a way to fix this?

I am using GitHub Copilot from the GitHub Education pack. My PyCharm version is PyCharm 2024.1.1
![Screenshot 2024-05-09 165315](https://github.com/community/community/assets/43165384/ec25981b-fa44-4186-b073-9610eaff98c3)
![Screenshot 2024-05-09 165224](https://github.com/community/community/assets/43165384/c6d84c3a-c3ae-413e-9a44-3e05aa1aa61a)

**Comment by techpond31** [comment]
👋Given your situation, it's possible that you reached the maximum conversation quota offered by GitHub Copilot during the trial period. Normally, after four conversational sessions, GitHub Copilot goes into cooldown mode, preventing you from starting new sessions or receiving suggestions for some time.

Since you obtained GitHub Copilot from the GitHub Education pack, I assume you're still eligible for the educational discount. To resume using GitHub Copilot, you can subscribe to the paid tier or extend the trial period if available.

To purchase a subscription, follow these steps:

1. Navigate to the JetBrains Plugins page in your web browser: https://plugins.jetbrains.com/plugin/16194-github-copilot
2. Click the Buy Now button or scroll down to the pricing table and pick a subscription plan.
3. Sign in to your JetBrains account, or create a new account if you haven't already.
4. Complete the payment process.

Once you've completed the subscription, you can continue using GitHub Copilot in PyCharm.

However, if you prefer extending the trial period or facing difficulty purchasing a subscription, I recommend reaching out to JetBrains Customer Service for assistance.

Contact JetBrains Customer Service:

1. Visit the JetBrains Help Center: https://www.jetbrains.com/support/
2. Fill out the Request Technical Assistance Form or submit a Sales Query Form, whichever suits your needs.

They'll help you sort out the issue and provide further guidance on getting GitHub Copilot up and running again.
